app/controllers/admin/form_templates_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Admin::FormTemplatesController < Admin::StaffController
+  before_action :ensure_form_templates_enabled
+
+  def index
+    form_templates = FormTemplate.all
+    render_serialized(form_templates, FormTemplateSerializer, root: "form_templates")
+  end
+
+  def new
+  end
+
+  def create
+    params.require(:name)
+    params.require(:template)
+
+    template = FormTemplate.new(name: params[:name], template: params[:template])
+
+    if template.save
+      render_serialized(template, FormTemplateSerializer, root: "form_template")
+    else
+      render_json_error(template)
+    end
```

Comment:
Any specific reason not to just use `create!` and rely on some default error handling?